lib/LaTeXML/Post/Graphics.pm

```diff
@@ -134,18 +134,22 @@ sub getGraphicsSourceTypes {
 sub findGraphicFile {
   my ($self, $doc, $node) = @_;
   if (my $source = $node->getAttribute('graphic')) {
-    # Find all acceptable image files, in order of search paths
-    my ($dir, $name, $reqtype) = pathname_split($source);
-    # Ignore the requested type? Or should it increase desirability?
-    #
-    # If this is *NOT* a known graphics type, it would be best to treat it as
-    # a name suffix (e.g. name.1 from name.1.png)
-    if ((length($reqtype) > 0) && !(grep { $_ eq lc($reqtype) } $$self{graphics_types})) {
-      $name .= ".$reqtype"; }
-    my $file  = pathname_concat($dir, $name);
-    my @paths = pathname_findall($file, paths => $LaTeXML::Post::Graphics::SEARCHPATHS,
-      # accept empty type, incase bad type name, but actual file's content is known type.
-      types => ['', $self->getGraphicsSourceTypes]);
+    # if we already have a usable candidate, save ourselves the work
```

Comment:
I also added two graphics-related upgrades:
 - If a graphic was available in the `candidates` attribute, we should use that path information in post-processing, which speeds things up along.
 - I used an example image distributed with texlive for the tests ( "example-image-a", it seemed appropriate), and realized we weren't finding those images. So added a fallback call to `pathname_kpsewhich` to Image.pm

Reply:
Hmm... using these images in the test cases is introducing a system-dependence that causes failure.

Reply:
Ah, I think this is due to the floating texlive directory name (based on texlive version). I suppose I should switch to the PNG I see in the manifest, `t/graphics/none.png` ?

But I think the `candidates` upgrade is actually healthy, especially in docs with lots of images (avoids a lot of duplicated searching)... Or?